source/_integrations/compensation.markdown

```diff
@@ -30,6 +30,20 @@ compensation:
     data_points:
       - [0.2, -80.0]
       - [1.0, 0.0]
+
+  media_player_db_volume:
+    source: media_player.yamaha_receiver_zone_2
+    attribute: volume_level
+    unit_of_measurement: dB
+    # Ensure that the sensor's value will not have a state lower than -80.0
+    # when the source sensors value is less than 0.2
+    apply_lower_limit: True
```

Comment:
```suggestion
    clamp_lower_limit: True
```

Reply:
I think there is a enough context to change everything to `lower_limit: True` and `upper_limit: True` which would make the config a bit simplier

Reply:
updated to `lower_limit`